Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -329,6 +330,49 @@ extension BrowserViewController: URLBarDelegate {
         hideSearchController()
         updateInContentHomePanel(tabManager.selectedTab?.url)
     }
+
+    func urlBar(urlBar: URLBarView, suggestionForText text: String, complete: String? -> Void) {
+        let options = QueryOptions(filter: text, filterType: FilterType.Url, sort: QuerySort.Frecency)
+        profile.history.get(options, complete: { (cursor: Cursor) in
```

Comment:
Oh, you're probably talking about the cursor for `SearchViewController`. The URL bar is completely decoupled from SVC right now, so that won't be straightforward, but I agree we should do that. I'll see if I can figure something out.